runners/katacoda/templates/npmInstall.md

```diff
@@ -2,9 +2,11 @@
 
 <%= cdCommand; %>
 
-Install the dependencies from the package.json file.
+<% if(packageName){ %>
+Install the package <%= packageName; %> and add it to the dependencies.<% } else {%>
+Install the dependencies from the package.json file.<% } %>
 
-`<% if(useDevonCommand){ %>devon<% } %> npm install`{{execute T1}}
+`<% if(useDevonCommand){ %>devon<% } %> npm install<% if(global){ %> -g <% } %><%= packageArgs; %> <%= packageName; %>`{{execute T1}}
```

Comment:
There is no space before the packageArgs argument. This will cause a problem when it is not installed globally.
And I think we should add a hint, what the -g parameter does.